src/utils/url.ts

```diff
@@ -38,33 +42,75 @@ export const adminCreateWorldSpace = (worldSlug?: string) =>
 export const adminWorldSpacesUrl = (worldSlug?: string) =>
   generatePath(ADMIN_V4_SPACES_1_PARAM_URL, { worldSlug });
 
```

Comment:
Plenty of callbacks for functions in here. Ideas are generally:
- encapsulate logic inside these utility functions otherwise their existence is mostly ceremonial
- don't introduce EMPTY_ constants, but instead check `isDefined` or `isTruthy` or similar
- probably use a single options argument that can easily be extended with named properties like `absolute?: boolean`
- shorter names starting with `generate` and ending with `Url`... generally... for quicker identification while scanning code
- possibly follow the `generate` with the core concept like `Space` or `World` then one more word for specialization

Reply:
Generated methods in a1a8ad822957ff7b46c3b7b881e87b113f27145c

Reply:
Generated methods in a1a8ad822957ff7b46c3b7b881e87b113f27145c